binr/radare2/radare2.c

```diff
@@ -527,6 +527,22 @@ int main(int argc, char **argv, char **envp) {
 		return 0;
 	}
 
+	// Try to set the correct scr.color for the current terminal.
```

Comment:
move all this logic into a separate function

Reply:
okay, assuming in the same file?

Reply:
hmm, somehow my compiler seems to throw away the function and pretend it doesn't exist, even though it's defined right above the main function.

Reply:
can you push it so we can have a look?

Reply:
will do